app/templating/summary/formatter_registry.py

```diff
@@ -0,0 +1,28 @@
+from app.templating.summary.formatters.check_box import CheckBoxFormatter
+from app.templating.summary.formatters.currency import CurrencyFormatter
+from app.templating.summary.formatters.date_range import DateRangeFormatter
+from app.templating.summary.formatters.radio_button import RadioButtonFormatter
+
+
+KNOWN_FORMATTERS = {
+    "GENERAL": {
+                'CHECKBOX': CheckBoxFormatter,
+                'CURRENCY': CurrencyFormatter,
+                'RADIO': RadioButtonFormatter,
+    },
+    "DATERANGE": {
+                'DATE': DateRangeFormatter,
+    },
+}
+
+
+class FormatterRegistry(object):
+
+    @staticmethod
+    def get_formatter(answer_type, question_type):
```

Comment:
Minor, `question_type` should be the first argument as it's the more significant parameter (the logic of the method uses it first and doesn't carry on if it doesn't match)